### PR TITLE
Egypt list update based on AFTE's report 

### PR DESCRIPTION
**Pull request by moeltaher**
Added AFTE's blocked websites list (97 websites) based on : https://afteegypt.org/right_to_know-2/publicationsright_to_know-right_to_know-2/2017/06/04/13069-afteegypt.html?lang=en

This brings the list to the most recent known state of blocking in Egypt

### CODE REVIEW

**Comment by agharbeia** [comment]
@anadahz, we've formed an adhoc collective of people from various countries and we'll be giving lists of Arab countries some attention.
So far updates are made to Palestine's and Egypt's lists

File: lists/eg.csv [hunk 1]
```diff
@@ -175,3 +175,88 @@ http://www.alarab.qa,NEWS,News Media,2017-05-24,OONI,blocked as of date added
...
+http://hasamegypt.com,MILX,Terrorism and Militants,2017-06-23,AFTE,
+http://ida2at.com,NEWS,News Media,2017-06-23,AFTE,
+http://institute.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://kh-press.com,NEWS,News Media,2017-06-23,AFTE,
+http://learning.aljazeera.net,NEWS,News Media,2017-06-23,AFTE,
+http://madaad.net,NEWS,News Media,2017-06-23,AFTE,
+http://marsadpress.net,NEWS,News Media,2017-06-23,AFTE,
+http://masralekhbaria.com,NEWS,News Media,2017-06-23,AFTE,
+http://masreiat.com,NEWS,News Media,2017-06-23,AFTE,
+http://medium.com,NEWS,News Media,2017-06-23,AFTE,
```
Comment (hellais) [inline lists/eg.csv]: this should probably land in the global list. also I believe medium.com has https, so the url should be specified with https://

**Comment by hellais** [comment]
This PR looks good to me. I am going to merge it and defer the https:// conversion/integration into the global list to later stage (we should probably do this for more URLs anyways).

And congrats on setting up the ad-hoc collective!

File: lists/eg.csv (hunk 1 above)
Comment (agharbeia) [inline lists/eg.csv]: @hellais, OK. Can you also explain to our working-group the criteria for inclusion in the gloal list? AlJazeera sites are probably an easy case. But what about the more obscure ones?
@moeltaher, can you take care of these comments before our next pull-request? Including the "https" part?

**Pull request merged**